src/tree/tree.scss

```diff
@@ -85,9 +85,9 @@ $iui-expander-button-width: ($iui-icons-default) + ($iui-expander-inline-padding
 
   &:focus {
     @include themed {
-      outline: thin solid rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-4));
```

Comment:
Why we can't use `iui-focus` mixin here?

Reply:
At first I thought this was a line that I missed, but I remember now that this was intentional.  The reason for this is because of the `&:focus:not(:focus-visible)` part on line 93.  The mixin sets `&:focus:not(:focus-visible) { outline: none; }` and in this specific case we need the outline set to -2px.  I could maybe update the mixin to support changing this.

Reply:
No, it's ok. I was just curious.